prow/cmd/tide/main.go

```diff
@@ -107,6 +107,9 @@ func main() {
 	if o.dryRun {
 		ghcSync = github.NewDryRunClient(oauthSecret, o.githubEndpoint.Strings()...)
 		ghcStatus = github.NewDryRunClient(oauthSecret, o.githubEndpoint.Strings()...)
+		if o.deckURL == "" {
```

Comment:
Ideally we refactor this so that it is unit-testable, like we've done in peribolos, etc